client/src/pages/Listings/Listings.tsx

```diff
@@ -25,9 +28,26 @@ const Listings = (): JSX.Element => {
     setDateRange(newDateRange);
   };
 
+  useEffect(() => {
+    try {
+      const fetchProfiles = async () => {
+        const fetchedProfiles = await getAllProfiles();
+        setProfiles(fetchedProfiles);
+      };
+      fetchProfiles();
+    } catch (error) {}
```

Comment:
Suggestion:

Maybe we can update the snackbar here if there was an error fetching the profiles?

Reply:
Applied